firmware/application/apps/ui_mictx.cpp

```diff
@@ -145,25 +145,25 @@ void MicTXView::rxaudio(bool is_on) {
         audio::input::stop();
         baseband::shutdown();
 
-        if (enable_am || enable_usb || enable_lsb || enable_dsb) {  // "NFM/FM",0 ," WFM  ",1 ,  "  AM  ",2, " USB  ", 3, " LSB  ",4, " DSB-SC", 5
```

Comment:
Ugh, I forgot I did this. I can put these changes in a second PR if anyone has concerns.
It's all just comment spacing/punctuation fixup.

I'll point out the real changes in comments below.